apps/dapp-console-api/docker-compose.yml

```diff
@@ -22,6 +33,8 @@ services:
     env_file: .env
     environment:
       - REDIS_URL=redis://redis:6379
+      - DB_HOST=postgres
```

Comment:
would set this to `dapp-console-api@oplabs-local-web.iam`, if you run into any connection errors from a fresh db run you'll most likely need to encode the `@`

Reply:
I think this needs to be set to `postgres` it's so the connection string points to the docker `postgres` service.